deployment/ccip/test_usdc_helpers.go

```diff
@@ -0,0 +1,222 @@
+package ccipdeployment
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/chainlink-ccip/pkg/reader"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/fee_quoter"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/mock_usdc_token_messenger"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/mock_usdc_token_transmitter"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/usdc_token_pool"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/shared/generated/burn_mint_erc677"
+	"math/big"
+)
+
+func ConfigureUSDCTokenPools(lggr logger.Logger, chains map[uint64]deployment.Chain, src, dst uint64, state CCIPOnChainState) (*burn_mint_erc677.BurnMintERC677, *burn_mint_erc677.BurnMintERC677, error) {
+	srcToken := state.Chains[src].BurnMintTokens677[USDCSymbol]
+	dstToken := state.Chains[dst].BurnMintTokens677[USDCSymbol]
+	srcPool := state.Chains[src].USDCTokenPool
+	dstPool := state.Chains[dst].USDCTokenPool
+
+	// Attach token pools to registry
+	if err := attachTokenToTheRegistry(chains[src], state.Chains[src], chains[src].DeployerKey, srcToken.Address(), srcPool.Address()); err != nil {
+		return nil, nil, err
```

Comment:
nit for this return and those below: does the `err` have enough context to debug or should we wrap with `fmt.Errorf`?